packages/concerto-core/lib/introspect/stringvalidator.js

```diff
@@ -60,14 +89,38 @@ class StringValidator extends Validator{
      */
     validate(identifier, value) {
         if(value !== null) {
-            if(!this.regex.test(value)) {
-                this.reportError(identifier, 'Value \'' + value + '\' failed to match validation regex: ' + this.regex);
+            if (this.regex) {
+                if(!this.regex.test(value)) {
+                    this.reportError(identifier, 'Value \'' + value + '\' failed to match validation regex: ' + this.regex);
```

Comment:
why not use string interpolation for this reportError?

Reply:
that's an existing code.